app/scripts/controllers/app-state.js

```diff
@@ -470,6 +470,14 @@ export default class AppStateController extends EventEmitter {
     });
   }
 
+  ///: BEGIN:ONLY_INCLUDE_IF(blockaid)
```

Comment:
We don't need this function, instead in the ppom middleware we can just do `appStateController.store.getState().signatureSecurityAlertResponses[securityAlertId]`